src/domain/schemas/lesson-schemas.js

```diff
@@ -0,0 +1,16 @@
+import joi from 'joi';
+import { idOrKeySchema } from './shared-schemas.js';
+import { slugSchema, sectionDBSchema } from './document-schemas.js';
```

Comment:
We should move those two schemas into `shared-schemas.js` as they are used in multiple places

Reply:
Done, thanks!

Reply:
When you move the `slugSchema` from `document-schemas.js` to `shared-schemas.js`, you also have to move the tests for `slugSchema` from `document-schemas.spec.js` to `shared-schemas.spec.js`.

Reply:
True, moved and added some section schema db tests.

Reply:
Cool, thanks